helper/rewrite.php

```diff
@@ -133,7 +146,16 @@ public static function isLocked() {
      */
     public static function addLock() {
         global $PLUGIN_MOVE_WORKING;
+        global $conf;
         $PLUGIN_MOVE_WORKING = $PLUGIN_MOVE_WORKING ? $PLUGIN_MOVE_WORKING + 1 : 1;
+        $lockfile = $conf['lockdir'] . 'move.lock';
```

Comment:
the lock file name would conflict with a lock for the page "move". I suggest using "_plugin_move.lock"